app/views/registrations/new.html.erb

```diff
@@ -26,6 +26,6 @@
     <%= f.password_field :password_confirmation %>
   </div>
   <div>
-    <%= f.submit "Register" %>
+    <%= f.submit "Submit" %>
```

Comment:
Good call. I changed it when I made my test and forgot to change it back.  I also could have wrote
within('.registration-form') { click_on 'Register'} 
I should have done that instead since I forgot to change submit back to register =)